data-prepper-plugins/geoip-processor/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation project(path: ':data-prepper-plugins:common')
+    implementation 'com.fasterxml.jackson.core:jackson-core'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'joda-time:joda-time:2.11.1'
```

Comment:
Please use Java 8 time instead of Joda time.

Reply:
Removed joda timedependency